### DISCUSSION TITLE
Explain command not showing output in databricks notebook

### DISCUSSION THREAD

**Discussion by harjeet-glean** [duckdb/duckdb#10680] [Q&A] [answered]
While running this command i can see the data , it took 15 minutes to get the data , I wanted to check whether partition push down is working, but I don't see output of explain

**Comment by szarnyasg** [comment]
Hi @harjeet-rzp, did you try these workarounds? https://duckdb.org/docs/api/python/known_issues#running-explain-renders-newlines-in-jupyter-and-ipython

**Reply by harjeet-glean** [reply, marked as answer]
checking
